source/styles/main.styl

```diff
@@ -126,7 +126,7 @@ nav
     padding 0 10px
     margin 0
     border-left 1px solid #DDD
-    transition all 0.2s
+    transition background 0.2s, color 0.2s, border-radius 0.2s
```

Comment:
It'd be better to move the `border-radius` under `a[href]` (or even just `a`) to avoid transitioning it altogether.